test/helpers/util.ts

```diff
@@ -1,7 +1,31 @@
-import { Request, request } from 'express';
+import { Request } from 'express';
 
-export const noop = async (): Promise<undefined> => undefined;
+import { RequestReader } from '../../src';
 
 export function createReq(props: unknown): Request {
-  return Object.assign(request, props) as Request;
```

Comment:
Assigning props to a library value was messing with unit tests as such value is not isolated. I fixed this in the new version.